pkg/codegen/internal/test/testdata/.sdkcodegenignore

```diff
@@ -0,0 +1,10 @@
+__pycache__
+bin
+node_modules
```

Comment:
This is kinda like poor-man's gitignore that excludes paths from the codegen diff-checking algo.